cdc_covidnet/delphi_cdc_covidnet/update_sensor.py

```diff
@@ -85,9 +85,15 @@ def update_sensor(
     ]
 
     # Set state id to two-letter abbreviation
-    geo_map = GeoMaps(static_path)
-    hosp_df = geo_map.state_name_to_abbr(hosp_df)
-
+    gmpr = GeoMapper()
+    hosp_df = gmpr.add_geocode(hosp_df,
+                               from_col=APIConfig.STATE_COL,
+                               from_code="state_name",
+                               new_code="state_id",
+                               dropna=False)
+    # To preserve column order, reassign original column and drop new one
```

Comment:
Is there a test that fails if this line is omitted?

Reply:
So if `hosp_df[APIConfig.STATE_COL] = hosp_df["state_id"].str.upper()` is omitted, tests fail because APIConfig.STATE_COL isn't the two letter abbreviation.

if `hosp_df.drop("state_id", axis=1, inplace=True)` is omitted, tests fail because dimensions are wrong.

I also actually just realized column order for these fields doesn't matter since there's a `hosp_df.set_index(["date", "geo_id"], inplace=True)` right after, so this is actually just to avoid name collisions since there isn't currently functionality to replace a column while keeping the name the same. I've updated the comment to reflect that.